homeassistant/components/cloud/strict_connection_guard_page.html

```diff
@@ -0,0 +1,140 @@
+        text-align: center;
+        margin-top: 32px;
+        margin-bottom: 32px;
+      }
+
+      .content {
+        box-sizing: border-box;
+        padding: 24px;
+        border-radius: 12px;
+        width: 90%;
+        max-width: 560px;
+        background-color: #ffffff;
+        border: 1px solid rgba(0, 0, 0, 0.12);
+        text-align: center;
+      }
+
+      .no-break {
+        white-space: nowrap;
+      }
+
+      h1,
+      h2 {
+        margin: 16px 0;
+        font-weight: normal;
+      }
+
+      h1 {
+        font-size: 28px;
+        line-height: 36px;
+      }
+
+      h2 {
+        font-size: 14px;
+        line-height: 20px;
+      }
+
+      .link {
+        display: inline-block;
+        margin-top: 8px;
+        padding: 12px 16px;
+        overflow: hidden;
+        border-radius: 4px;
+        text-transform: uppercase;
+        text-decoration: none;
+        font-weight: 500;
+        font-size: 12.25px;
+        letter-spacing: 1.09375px;
+        background-color: rgb(3, 169, 244);
+        color: white;
+        transition: background-color 0.3s;
+      }
+
+      .link:hover {
+        background-color: rgba(3, 169, 244, 0.8);
+      }
+
+      .link:focus-visible {
+        background-color: rgba(3, 169, 244, 0.7);
+      }
+
+      .link:active {
+        background-color: rgba(3, 169, 244, 0.6);
+      }
+    </style>
+  </head>
+
+  <body>
+    <div class="header">
+      <svg
+        width="56"
+        height="56"
+        viewbox="0 0 240 240"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <path
+          d="M240 224.762c0 8.25-6.75 15-15 15H15c-8.25 0-15-6.75-15-15v-90c0-8.25 4.77-19.769 10.61-25.609l98.78-98.78c5.83-5.83 15.38-5.83 21.21 0l98.79 98.789c5.83 5.83 10.61 17.36 10.61 25.61v90-.01z"
+          fill="#F2F4F9"
+        />
+        <path
+          d="M229.39 109.153l-98.78-98.78c-5.83-5.83-15.38-5.83-21.21 0l-98.79 98.78C4.78 114.983 0 126.512 0 134.762v90c0 8.25 6.75 15 15 15h92.27l-40.63-40.63c-2.09.72-4.32 1.13-6.64 1.13-11.3 0-20.5-9.2-20.5-20.5s9.2-20.5 20.5-20.5 20.5 9.2 20.5 20.5c0 2.33-.41 4.56-1.13 6.65l31.63 31.63v-115.88c-6.8-3.34-11.5-10.32-11.5-18.39 0-11.3 9.2-20.5 20.5-20.5s20.5 9.2 20.5 20.5c0 8.07-4.7 15.05-11.5 18.39v81.27l31.46-31.46c-.62-1.96-.96-4.04-.96-6.2 0-11.3 9.2-20.5 20.5-20.5s20.5 9.2 20.5 20.5-9.2 20.5-20.5 20.5c-2.5 0-4.88-.47-7.09-1.29L129 208.892v30.88h96c8.25 0 15-6.75 15-15v-90c0-8.25-4.77-19.77-10.61-25.61v-.009z"
+          fill="#18BCF2"
+          alt="Home Assistant logo"
+        />
+      </svg>
+    </div>
+    <div class="content">
+      <h1>You need access</h1>
+      <h2>
+        This device is not known to
+        <span class="no-break">Home Assistant</span>.
+      </h2>
+
+      <a
+        target="_blank"
+        rel="noreferrer noopener"
+        class="link"
+        href="https://nabucasa.com/config/remote/#to-gain-remote-access-from-a-new-device-when-strict-connection-is-enabled"
```

Comment:
if the difference is just the URL, why not just put `{{ url }}` placeholder in the guard page HTML and replace on the fly?